src/App.jsx

```diff
@@ -14,57 +16,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [message, setMessage] = useState('Jam is selected');
```

Comment:
The initial state for `selectedGood` and `message` is set to 'Jam' and 'Jam is selected', respectively. While this works, it might be clearer to initialize these states to an empty string or null and update them based on user interaction. This would make the initial state more consistent with the logic used when clearing the selection.